### DISCUSSION TITLE
Allow editing repo path/url?

### DISCUSSION THREAD

**Discussion by jcroy** [garethgeorge/backrest#392] [Ideas]
I'm backing up to an external disk using backrest. Works great. Only issue is that because the disk path change and now I have to setup the repo again. 
Now I could prevent this by setting  the disk drive letter to a higher value. People who backup to external disks, are you doing the forementioned?

**Comment by berti34** [comment]
For a quick workaround you can stop the backrest service and edit the config.json
Restart the backrest service and the REPO config has changed.

**Reply by berti34** [reply]
On Windows I'm using the cli tool mountvol to get the ID.
For example  "mountvol E:"  shows
\\\\?\\Volume{279b9f93-0000-0000-0000-100000000000}\\

So I can use the string   \\\\?\\Volume{279b9f93-0000-0000-0000-100000000000}\\restic\\  
as complete path to the repository on an usb disk, regardless of the letter. Works very well.

Greetings

**Reply by garethgeorge** [reply]
+1 re: this option and editing the config directly as the supported workaround.
